DigitalLearningSolutions.Data/Services/RegistrationService.cs

```diff
@@ -169,5 +193,38 @@ bool refactoredTrackingSystemEnabled
 
             return new Email(emailSubject, body, emailAddress);
         }
+
+        private Email GenerateWelcomeEmail(
+            string emailAddress,
+            string firstName,
+            string lastName,
+            string centreName,
+            string candidateNumber,
+            string setPasswordHash
+        )
+        {
+            const string emailSubject = "Welcome to Digital Learning Solutions - Verify your Registration";
+            var setPasswordUrl = $"{config["AppRootPath"]}/SetPassword?code={setPasswordHash}&email={emailAddress}";
+            
+            BodyBuilder body = new BodyBuilder
+            {
+                TextBody = $@"Dear {firstName} {lastName},
+                            An administrator has registered your details to give you access to the Digital Learning Solutions (DLS) platform under the centre {centreName}.
+                            You have been assigned the unique DLS delegate number {candidateNumber}.
+                            To complete your registration and access your Digital Learning Solutions content, please follow this link:{setPasswordUrl}
+                            Note that this link can only be used once.
+                            Please don't reply to this email as it has been automatically generated.",
+                HtmlBody = $@"<body style= 'font - family: Calibri; font - size: small;'>
+                                <p>Dear {firstName},</p>
+                                <p>An administrator has registered your details to give you access to the Digital Learning Solutions (DLS) platform under the centre {centreName}.</p>
+                                <p>You have been assigned the unique DLS delegate number {candidateNumber}.</p>
+                                <p>To complete your registration and access your Digital Learning Solutions content, please follow this link: <a href=""{setPasswordUrl}"">{setPasswordUrl}</a></p>

```

Comment:
How does this hyperlink read out with a screen reader? Would it be better to name it something more clear like "Click here to complete your registration..."

We have a little bit of wiggle room to change things here to be more accessible :)